master/custom/factories.py

```diff
@@ -893,7 +893,9 @@ def setup(self, parallel, branch, test_with_PTY=False, **kwargs):
                 warnOnFailure=True,
             )
         )
-        self.host_configure_cmd.append(util.Interpolate("CONFIG_SITE=%(prop:config_site)s"))
+        self.compile_environ.update(
+            CONFIG_SITE=util.Interpolate("%(prop:config_site)s")
```

Comment:
Also need to add the path prefix:
```suggestion
            CONFIG_SITE=util.Interpolate("../../%(prop:config_site)s")
```